src/lib/code-sample/schema.ts

```diff
@@ -37,7 +38,7 @@ export type CodeSampleDefinition = z.output<typeof CodeSampleDefinitionSchema>
 
 const CodeSampleSchema = CodeSampleDefinitionSchema.extend({
   code: z.record(
-    z.enum(['javascript', 'python', 'php', 'ruby']),
+    z.enum(['javascript', 'python', 'php', 'ruby', 'bash']),
     z.object({
       title: z.string().min(1),
```

Comment:
Let's add a new enum to pass though to the markdown code block,

```ts
const syntax = z.enum(['javascript', 'python', 'php', 'ruby', 'bash'])
```

```suggestion
      title: z.string().min(1),
      request_syntax: syntax,
      response_syntax: syntax,
```

Reply:
updated in https://github.com/seamapi/blueprint/pull/58/commits/0b1f37a41acad1ce4a296178b333d1b07f8c20b7